File: SpringBoot_Backend/src/main/java/net/bergouIbrahim/springboot/model/employee.java
package net.bergouIbrahim.springboot.model;


import com.fasterxml.jackson.annotation.JsonProperty;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "Employee")
public class employee {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ID")
    @JsonProperty("id")
    private Integer id;

    @Column(name = "First_Name")
    @JsonProperty("firstname")
    private String firstname;

    @Column(name = "Last_Name")
    @JsonProperty("lastname")
    private String lastname;

    @Column(name = "Email")
    @JsonProperty("email")
    private String email;
}

File: Angular_Frontend/src/app/employee.ts
export class Employee {

    id?: number;
    firstname?: string;
    lastname?: string;
    email?: string;

    constructor(id?:number,first_name?:string,last_name?:string,email?:string){

        this.id=id;
        this.firstname=first_name;
        this.lastname=last_name;
        this.email=email;

    }
}
